src/main/java/me/illia/jlog/JLogger.java

```diff
@@ -1,53 +1,104 @@
 package me.illia.jlog;
 
-import java.util.Random;
+import me.illia.jlog.log.LogFile;
+import me.illia.jlog.other.JLogColor;
+import me.illia.jlog.other.JLogTimestamp;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Scanner;
 
 public class JLogger {
-    public static JLoggerStyle style;
-    // The return type of the log() method represents did the program fail, did it succeed, or did it 'super failed' aka critical error
-    public static int log(JLogLvl lvl, Object msg) {
-        int result = -2;
-
-        if (style == null) {
-            result = -1;
-            throw new IllegalArgumentException("You should call the method 'setStyle' before logging anything to the console!");
-        }
+    private static final Scanner IN = new Scanner(System.in);
+    public static Date date = new Date();
```

Comment:
This will probably fail to set the correct date when logger is created and then you log some message after 0:00

Reply:
Oh, yeah, but where do i create it then? Do i "setDate()" in the "Demo"?